x/ccv/types/errors.go

```diff
@@ -25,4 +25,5 @@ var (
 	ErrClientNotFound           = errorsmod.Register(ModuleName, 18, "client not found")
 	ErrDuplicateConsumerChain   = errorsmod.Register(ModuleName, 19, "consumer chain already exists")
 	ErrConsumerChainNotFound    = errorsmod.Register(ModuleName, 20, "consumer chain not found")
+	ErrInvalidEvidence          = errorsmod.Register(ModuleName, 21, "invalid consumer double voting evidence")
```

Comment:
Would it make sense to call this `ErrInvalidDoubleVotingEvidence`?

From what I understand, we do not have an error type for misbehaviour because you use IBC error types such as `ibcclienttypes.ErrInvalidMisbehaviour` already?